pkg/crc/preflight/preflight_checks_linux.go

```diff
@@ -22,14 +22,15 @@ import (
 )
 
 const (
-	driverBinaryDir      = "/usr/local/bin"
-	kvmDriverBinaryPath  = driverBinaryDir + "/docker-machine-driver-kvm"
-	kvmDriverDownloadURL = "https://github.com/dhiltgen/docker-machine-kvm/releases/download/v0.10.0/docker-machine-driver-kvm-centos7"
-	defaultPoolSize      = "20 GB"
-	nodeMac              = "52:fd:fc:07:21:82"
-	nodeIp               = "192.168.126.11"
-	domName              = "crc"
-	poolTemplate         = `<pool type='dir'>
+	driverBinaryDir          = "/usr/local/bin"
+	libvirtDriverCommand     = "crc-driver-libvirt"
+	libvirtDriverBinaryPath  = driverBinaryDir + libvirtDriverCommand
```

Comment:
This should go as variable or and we should use `filepath.Join(driverBinaryDir, libvirtDriverCommand)` since I was having a hard time to debug why setup is not successfully working and found out this string provide `/usr/local/bincrc-driver-libvirt` without a forward slash and interestingly there is no error message till end :(

Reply:
ah right... `filepath.Join` ;-D hahahaa

Reply:
so logging needs to be addressed for the pre-flight checks